src/main/webapp/signup.html

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<meta charset="UTF-8">
+<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+<title>Sign Up</title>
+<!-- Bootstrap CSS only -->
+<link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css" integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk" crossorigin="anonymous">
+<!-- Custom styles -->
+<link rel="stylesheet" href="style.css">
+
+<div class="container">
+  <h1 class="display-4 mt-5">Sign Up for Open Sesame!</h1>
+  <h4 class="emphasis mb-5">Looking to find a mentor within the open source community, or become a mentor to foster the next generation of open source contributors? Sign up here!</h4>
+  <hr/>
+  <form>
+    <div class="form-group">
+      <button class="font-weight-bold">Link Your GitHub Account</button>
+      <small id="gitHubHelp" class="form-text text-muted">We will use your GitHub account to populate your name and bio, authenticate you, and message you when you match with a mentor/mentee.</small>
```

Comment:
After switch to Users API :(
```suggestion
      <small id="gitHubHelp" class="form-text text-muted">We will use your GitHub account to populate your name and bio and message you when you match with a mentor/mentee.</small>
```

Reply:
Sadness.